Makefile

```
@@ -360,6 +360,9 @@ ifdef LLAMA_METAL
 	MK_CPPFLAGS += -DGGML_USE_METAL
 	MK_LDFLAGS  += -framework Foundation -framework Metal -framework MetalKit
 	OBJS		+= ggml-metal.o
+ifdef LLAMA_METAL_NDEBUG
+	MK_CPPFLAGS += -DGGML_METAL_NDEBUG
```

```suggestion
	MK_CFLAGS += -DGGML_METAL_NDEBUG
```

It should be used for ggml-metal.m.

---

cpp is the C preprocessor - CPPFLAGS is used for both C and C++ compilation.

---

Yeah i think i misunderstood, sorry about that.